### DISCUSSION TITLE
Issue with 230x230

### DISCUSSION THREAD

**Discussion by dwidou** [Jyers/Marlin#499] [Q&A] [answered]
Hi Jyers,
I can't print a complete 230x230 bed size. I only can print 230 (X) x 227 (Y). To do that I offset X -2mm and Y -8mm. If I st Y offset to -11mm, it doesn't change because Y endstop.
I try to set origin offset in g-code simplify3d, or try to set toolhead offset, but those 2 offsets doens't change anything.
Only "global G-code offsets" is working, but always issue with Y endstop.

How can I really print 230x230 ?

**Comment by Jyers** [comment, marked as answer]
I'm assuming you are using a print head configuration where the nozzle is not on the bed at the endstops. In this case there is not way for you to print 230x230 without compiling your own version. The configuration is set up to have stops at 230x230 no matter the offset, so you would need to modify the bed size in your configuration file.
